docs/get-started/table-theme-options.qmd

```diff
@@ -0,0 +1,203 @@
+
+The table body can style the lines between individual cells. Use the `hline` and `vline` option types to specify cell line color, style, and width.
+
+For example, the code below changes horizontal lines (`hline`) between cells to be red, dashed lines.
+
+```{python}
+gt_ex.tab_options(
+    table_body_hlines_color="red",
+    table_body_hlines_style="dashed",
+    table_body_hlines_width="4px",
+)
+```
+
+In order to define the vertical lines between cells, set `vline` styles. For example, the code below makes both horizontal and vertical lines between cells solid.
+
+
+```{python}
+gt_ex.tab_options(
+    table_body_hlines_style="solid",
+    table_body_vlines_style="solid",
+)
+```
+
+## Set options across table parts
+
+Some options starting with `table_` apply to all parts of the table.
+For example, fonts and background color apply everywhere.
+
+```{python}
+
+gt_ex.tab_options(
+    table_background_color="green",
+    table_font_color="darkblue",
+    table_font_style="italic",
+    table_font_names="Times New Roman"
+)
+```
+
+Options set across the whole table, can be overriden by styling a specific part.
+
+```{python}
+gt_ex.tab_options(
+    table_background_color="orange",
+    heading_background_color="pink"
+)
+```
+
+## A basic theme
+
+Based on the sections above, we can design an overall theme for a table.
+
+This requires setting a decent number of options, but makes a big difference when presenting a table! Below is a table with a simple, blue theme. (The code is hidden by default, but can be expanded to see all the options set).
+
+```{python}
+#| code-fold: true
+from great_tables import GT, exibble
+
+# TODO: are there names we can give the three colors?
+# e.g. primary = "#0076BA", etc..
+
+(GT(exibble, rowname_col="row", groupname_col="group")
+    .tab_header("THE HEADING", "(a subtitle)")
+    .tab_stubhead("THE STUBHEAD")
+    .tab_source_note("THE SOURCE NOTE")
+    .tab_options(
+        # table ----
+        table_border_top_color="#004D80",
+        table_border_bottom_color="#004D80",
+
+        # heading ----
+        heading_border_bottom_color="#0076BA",
+
+        # column labels ----
+        column_labels_border_top_color="#0076BA",
+        column_labels_border_bottom_color="#0076BA",
+        column_labels_background_color="#FFFFFF",
+
+        # row group ----
+        row_group_border_top_color="#0076BA",
+        row_group_border_bottom_color="#0076BA",
+
+        # stub ----
+        stub_background_color="#0076BA",
+        stub_border_style="solid",
+        stub_border_color="#0076BA",
+
+        # table body ----
+        table_body_border_top_color="#0076BA",
+        table_body_border_bottom_color="#0076BA",
+        table_body_hlines_style="none",
+        table_body_vlines_style="none",
+
+        # misc ----
+        #row_striping_background_color="#F4F4F4"
+    )
+
+)
+```
+
+## Composite options
+
+TODO: which options are handled in a special way? (e.g. `row_group_as_column = True`)
```

Comment:
Any idea which options do a set of things, rather than a single thing? (Or which of those options would be good to illustrate---either here or in their own section).

Reply:
container_width and container_height (and container_overflow_x|y) do more complex things which are worth illustrating (especially if space is at a premium for the table on a page).